fhir-search/src/main/java/com/ibm/fhir/search/SearchConstants.java

```diff
@@ -140,21 +140,30 @@ private SearchConstants() {
                 private static final long serialVersionUID = -7809685447880880523L;
 
                 {
-                    put(SearchConstants.Type.STRING, Arrays.asList(SearchConstants.Modifier.EXACT, SearchConstants.Modifier.CONTAINS, SearchConstants.Modifier.MISSING));
-                    put(SearchConstants.Type.REFERENCE, Arrays.asList(SearchConstants.Modifier.TYPE, SearchConstants.Modifier.MISSING));
-                    put(SearchConstants.Type.URI, Arrays.asList(SearchConstants.Modifier.BELOW, SearchConstants.Modifier.MISSING, SearchConstants.Modifier.ABOVE));
-                    put(SearchConstants.Type.TOKEN, Arrays.asList(SearchConstants.Modifier.BELOW, SearchConstants.Modifier.NOT, SearchConstants.Modifier.MISSING));
-                    put(SearchConstants.Type.NUMBER, Arrays.asList(SearchConstants.Modifier.MISSING));
-                    put(SearchConstants.Type.DATE, Arrays.asList(SearchConstants.Modifier.MISSING));
-                    put(SearchConstants.Type.QUANTITY, Arrays.asList(SearchConstants.Modifier.MISSING));
+                    put(SearchConstants.Type.STRING, Arrays.asList(Modifier.EXACT, Modifier.CONTAINS, Modifier.MISSING));
+                    put(SearchConstants.Type.REFERENCE, Arrays.asList(Modifier.TYPE, Modifier.MISSING));
+                    put(SearchConstants.Type.URI, Arrays.asList(Modifier.BELOW, Modifier.ABOVE, Modifier.MISSING));
+                    put(SearchConstants.Type.TOKEN, Arrays.asList(Modifier.BELOW, Modifier.NOT, Modifier.MISSING));
+                    put(SearchConstants.Type.NUMBER, Arrays.asList(Modifier.MISSING));
+                    put(SearchConstants.Type.DATE, Arrays.asList(Modifier.MISSING));
+                    put(SearchConstants.Type.QUANTITY, Arrays.asList(Modifier.MISSING));
+                    put(SearchConstants.Type.COMPOSITE, Arrays.asList(Modifier.MISSING));
```

Comment:
this might be where a conflict comes up... with 'near'

Reply:
Actually there no conflict on this part since Location search doesn't currently support any modifiers.
When we get to #510, we'll need to add Type.SPECIAL to this list with a singleton of `Modifier.MISSING`.